### ISSUE TITLE
Report list of new hits for every ORF.

### ISSUE THREAD

**Issue by ilectra** [UCL/HHyeast-server#12]

**Comment by tamuri** [comment]
Does this mean highlighting new hits for the ORF that are not in SGD’s domain list?

**Comment by timlevine** [comment]
yes - but not literally, as the precise identification is so diifcult.

more like:
reporting where we get strong hits in regions that had no Pfam hit:
this overall report should have multiple versions:

one for new PFAM hits where there was no pfam hit before
one for new PDB hits        "         "        "        "        "        "
one for new yeast hits (other than full-length)         "        "        "        "        "        "

and for each of these we should record the overall "Prob" (what i call pSS) so that we can make a table relating the level of confidence for the new hits to their number/extent

Is that OK?
________________________________
